.github/workflows/ci.yml

```diff
@@ -8,25 +8,25 @@ on:
       - master
 jobs:
   build:
+    runs-on: ${{ matrix.os }}
+    if: true
     strategy:
       fail-fast: false
       matrix:
         os:
-          - macos-latest
           - ubuntu-latest
-          - windows-latest
+          - macos-latest
+          # - windows-latest
```

Comment:
So we do not test GeneWeb at all on Windows?

Reply:
We do, we do, we just need to rewrite a last clean commit with a synthesis of what's wanted.

Reply:
(Those if: true/false/or OS removal were done to shorten the various test builds that came afterwards, the first one took hours with a matrix of 40 builds! #save the planet)